ui/src/lib/gql/queries/user.ts

```diff
@@ -16,13 +15,14 @@ graphql(`
   }
 `);
 
+export enum UserQueryKeys {
+  GetMe = "getMe",
+}
```

Comment:
Is this not available from the generated code?

Reply:
No, not that I can could find. The other plugin had it but it doesn't work with v5. 
ref: 
> [typescript-react-query](https://the-guild.dev/graphql/codegen/plugins/typescript/typescript-react-query) would be better but it's not working with v5 at the moment . Link to the [issue](https://github.com/dotansimha/graphql-code-generator-community/issues/586)

Reply:
Ah yeah this was one of the "we will improve it when we can" things. Gotcha